app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -279,3 +299,110 @@ private fun FavoriteParkingCard(
+          itemsIndexed(userReviews) { _, review -> review?.let { ReviewCard(review = it) } }
+        }
+  }
+}
+
+@Composable
+private fun ReviewCard(review: Review) {
+  Card(modifier = Modifier.padding(8.dp).width(260.dp), shape = MaterialTheme.shapes.medium) {
+    Column(modifier = Modifier.padding(16.dp)) {
+      Text(
+          text = review.parking,
+          style = MaterialTheme.typography.titleMedium,
+          modifier = Modifier.fillMaxWidth())
+
+      Spacer(modifier = Modifier.height(8.dp))
+
+      Text(
+          text =
+              stringResource(
+                  R.string.view_profile_screen_you_rated_parking, review.rating.toString()),
+          style = MaterialTheme.typography.bodyMedium,
+          color = MaterialTheme.colorScheme.primary)
+
+      Spacer(modifier = Modifier.height(8.dp))
+
+      Text(
+          text = stringResource(R.string.view_profile_screen_you_said),
+          style = MaterialTheme.typography.bodyMedium,
+          color = MaterialTheme.colorScheme.onSurfaceVariant,
+          modifier = Modifier.testTag("YouSaidText_${review.uid}"))
+
+      Text(
+          text = "\"${truncateText(review.text)}\"",
+          style = MaterialTheme.typography.bodyMedium,
+          modifier = Modifier.fillMaxWidth())
+
+      Spacer(modifier = Modifier.height(8.dp))
+
+      Row(
+          modifier = Modifier.fillMaxWidth(),
+          horizontalArrangement = Arrangement.End,
+          verticalAlignment = Alignment.CenterVertically) {
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(12.dp),
+                verticalAlignment = Alignment.CenterVertically) {
+                  Row(
+                      horizontalArrangement = Arrangement.spacedBy(4.dp),
+                      verticalAlignment = Alignment.CenterVertically,
+                      modifier = Modifier.testTag("LikesCount_${review.uid}")) {
+                        Text(
+                            text =
+                                stringResource(
+                                    R.string.view_profile_screen_likes_count, review.likedBy.size),
+                            style = MaterialTheme.typography.bodyMedium)
+                      }
+                  Row(
+                      horizontalArrangement = Arrangement.spacedBy(4.dp),
+                      verticalAlignment = Alignment.CenterVertically,
+                      modifier = Modifier.testTag("DislikesCount_${review.uid}")) {
+                        Text(
+                            text =
+                                stringResource(
+                                    R.string.view_profile_screen_dislikes_count,
+                                    review.dislikedBy.size),
+                            style = MaterialTheme.typography.bodyMedium)
+                      }
+                }
+          }
+    }
+  }
+}
+
+private fun truncateText(text: String, maxLength: Int = 65): String {
```

Comment:
Wow, that's a super creative way of doing that, nice! maybe the max length could be a const val?

Reply:
Im not sure I did what you wanted : 
- change the parameter to the const val (not what i did)
- remove the parameter and use the const val inside the function (what i did)

Reply:
What you did works just fine!